app/assets/stylesheets/application.scss

```diff
@@ -12,7 +12,4 @@
  *
  *= require_tree .
  *= require_self
- */
-
-@import "bootstrap-sprockets";
-@import "bootstrap";
\ No newline at end of file
+ */
```

Comment:
Please add a line break at the end of the file.

Reply:
Yes, I've edited